api-reference/_hidden/dxDataGridColumn/groupCellTemplate.md

```diff
@@ -60,6 +60,108 @@ The group cell's value with applied [format](/api-reference/_hidden/dxDataGridCo
 The group cell's value as it is specified in a data source.
 
 ---
+
+Group cells display the following data:
+
+* The caption and the value of the column by which data is grouped;
+
+* [Group summary items]
+
+* [groupContinuesMessage] and [groupContinuedMessage] (if the group is spread across two or more pages)
+
+The following example shows how to display column value only:
+
+---
+##### jQuery
+
+    <!--JavaScript-->
+    $(function() {
+        $("#dataGridContainer").dxDataGrid({
+            // ...
+            columns: [ 
+                {
+                    dataField: "Country",
+                    groupIndex: 0,
+                    groupCellTemplate: function(element, options) {
+                        element.text(options.value);
+                    }
+                },
+            // ...
+            ]
+        });
+    });
+
+##### Angular
+    
+    <!--HTML-->
+    <dx-data-grid ... >
+        <dxi-column 
+            dataField="Country" 
+            [groupIndex]="0" 
+            groupCellTemplate="groupCellTemplate">
+        </dxi-column>
+
+        <div *dxTemplate="let data of 'groupCellTemplate'">
+    	    <div>{{data.value}}</div>
+  	    </div>
+    </dx-data-grid>
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid ... >
+            <DxColumn
+                data-field="Country"
+                :group-index="0"
+                group-cell-template="groupCellTemplate"
+            />
+
+            <template #groupCellTemplate="{ data }">
+                <div>{{data.value}}</div>
+            </template>
+        </DxDataGrid>
+    </template>
+    <script>
+    import DxDataGrid from 'devextreme-vue/data-grid';
+
+    export default {
+        components: {
+            DxDataGrid
+        },
+        // ...
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import {DataGrid} from 'devextreme-react/data-grid';
+
+    export default function App() {
+        // ...
+        return (
+            <DataGrid ...>
+                <Column
+                    dataField={'Country'}
+                    defaultGroupIndex={0}
+                    groupCellRender={GroupCell}/>
+            </DataGrid>
+        );
+    }
+
+    function GroupCell(options) {
+        let displayValue = options.value;
+        return (
+            <div>{displayValue}</div>
+        );
+    }
+
+    export default App;
```

Comment:
App is already exported at [line 142](https://github.com/DevExpress/devextreme-documentation/pull/1452/files#diff-c4463ca3296cf28f8e694f00a4fd1ff4594df8f017b6014f2a4e9267157a8563R142). Please ensure that code samples work before submitting them.